supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -531,3 +562,14 @@ Do not use "BIOS" as a generic term to refer to computer firmware. Use "firmware
 *Incorrect forms*: byte code
 
 *See also*:
+
+[discrete]
+[[byte-compiled-program]]
+==== image:images/yes.png[yes] byte-compiled program (noun)
+*Description*: _Byte-compiled programs_ are programs that need to be compiled into byte code, which is then executed by a language virtual machine.
```

Comment:
--> ... are programs that must be compiled into byte code

change "executed" to "started"?

Reply:
IBM Style: "Execute: Use only if a simpler term, such as run, issue, start, or enter is inappropriate."

Reply:
```suggestion
*Description*: _Byte-compiled programs_ are programs that need to be compiled into byte code before running in a language virtual machine.
```

Reply:
```suggestion
*Description*: _Byte-compiled programs_ are programs that must be compiled into byte code before they can run in a language virtual machine.
```